src/java.desktop/share/classes/javax/swing/CellRendererPane.java

```diff
@@ -200,7 +200,8 @@ public void paintComponent(Graphics g, Component c, Container p, int x, int y, i
 
 
     /**
-     * Calls this.paintComponent() with the rectangles x,y,width,height fields.
+     * Calls this.paintComponent(g, c, p, x, y, w, h) with the input parameter
+     * {@code Rectangle} x,y,width,height fields.
```

Comment:
Probably the simple "Calls paintComponent(g, c, p, r.x, r.y, r.width, r.height)" will work?

Reply:
Then the question might arise what is r.x, r.y....I guess my wordings makes it more clearer with pointer to input paramter.

Reply:
Can this please be approved if there's no more comments?

Reply:
> Probably the simple "Calls paintComponent(g, c, p, r.x, r.y, r.width, r.height)" will work?

…where `{@code r}` is the passed in rectangle.

I'd add spaces after commas: x, y, width, height.

Reply:
OK. done